.github/workflows/integration.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   integration:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Please provide an explanation to the commit message why this is the correct change.

Reply:
Updated.